dz4/z2.py

```diff
@@ -0,0 +1,11 @@
+# Напишите функцию для транспонирования матрицы
+
+
+def trans_matr(matr):
+    zip_rows = zip(*matr)
+    trans = [list(row) for row in zip_rows]
+    return trans
```

Comment:
Можно проще и в одну строку при этом строка совсем не сложная.
```suggestion
    return [*map(list, zip(*matr))]
```